ophyd/utils/epics_pvs.py

```diff
@@ -428,11 +428,12 @@ def _compare_maybe_enum(a, b, enums, atol, rtol):
 
 
 _type_map = {'number': (float, ),
-             'array': (np.ndarray, ),
+             'array': (np.ndarray, list),
```

Comment:
I guess this is fine, but I have a feeling it was working around what `ArrayAttributeSignal` was made for: https://github.com/NSLS-II/ophyd/blob/master/ophyd/signal.py#L941